src/lib/config.js

```diff
@@ -119,11 +119,28 @@ async function config() {
       },
       result: (val) => val.trim().replace(/ /, ''),
     },
+    {
+      type: 'input',
+      name: 'fee',
+      message: (state) => {
+        const style = state.submitted && !state.cancelled ? green : reset;
+        return style(
+          'Set default transaction fee to deploy a zkApp (in MINA):'
```

Comment:
we'd want to use 'smart contract', instead of a zkApp. But maybe not needed in this context?

Context:
```
✔ Choose a name (can be anything): · testnet
✔ Set URL to deploy to: · https://api.example.com
```
Maybe this is sufficient? It is hard to phrase :)
`Set transaction fee to use when deploying (in MINA):`